package.json

```diff
@@ -48,20 +48,24 @@
     "electron-log": "4.3.2",
     "electron-squirrel-startup": "1.0.0",
     "electron-store": "5.1.1",
+    "emoji-mart": "^3.0.1",
     "hex-string": "1.0.3",
     "history": "4.10.1",
     "http": "0.0.1-security",
+    "immer": "9.0.2",
     "ini": "2.0.0",
     "javascript-terminal": "1.1.1",
     "js-base64": "3.5.2",
     "js-sha256": "0.9.0",
     "libsodium-wrappers-sumo": "0.7.6",
+    "moment": "2.29.1",
```

Comment:
We already have `[day.js](https://day.js.org/)`, I think it's pretty much a replacement for `moment`